src/transformers/configuration_blenderbot.py

```diff
@@ -0,0 +1,117 @@
+import os
+
+from transformers.configuration_bart import BartConfig
+
+
+BLENDERBOT_PRETRAINED_CONFIG_ARCHIVE_MAP = {"blenderbot": "sshleifer/blenderbot-3B"}
+
+
+class BlenderbotConfig(BartConfig):
```

Comment:
Do we have to abstract the config from `BartConfig`? I'm very much against this because we don't gain anything really from it IMO and now looking it is not enough to just look at the `BlenderbotConfig`, one also always has to check `BartConfig`. It is not straight-forward to see here which Bart Config params are overwritten and which are not. 
The configs don't have any functionality that is reused here except avoiding to rewrite these `self...` statements. Don't like this too much here.

Reply:
Pinging @sshleifer and @LysandreJik to hear their opinion on that